app/jobs/runtime/delete_expired_package_blob.rb

```diff
@@ -17,11 +17,7 @@ def perform
           package = PackageModel.find(guid: package_guid)
           return unless package
 
-          if VCAP::CloudController::Config.config.package_image_registry_configured?
-            VCAP::CloudController::Jobs::Kubernetes::RegistryDelete.new(package.bits_image_reference).perform
-          else
-            BlobstoreDelete.new(package_guid, :package_blobstore).perform
-          end
+          create_package_source_deletion_job(package)&.perform
```

Comment:
Is the & here marking this as a block that gets passed to perform or... something else?

I'm looking at (this article on the ampersand operator)[https://medium.com/@sologoubalex/parameter-with-ampersand-operator-in-ruby-6a6a7fd666d5] but it doesn't look like an exact match for this syntax.

Reply:
This is the "safe navigation" operator that was introduced in Ruby 2.3 (I think).

https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/

An alternative way of writing this would be:
```ruby
package_src_delete_job = create_package_source_deletion_job(package)
package_src_delete_job.perform if package_src_delete_job
```

But I liked the way this read and felt very Rubyish. 😂 

I can change it to a more explicit form if you'd like.

Reply:
safe cuz it's nil-safe :)